scripts/utils/helper.js

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
utils are treated as generic helpers, having said that never create constant.js or helper.js inside utils folder

inside utils your function name becomes file name and it should be default exported from the file

Reply:
ok